sass/parts/_header.scss

```diff
@@ -1,77 +1,112 @@
 .page-header {
-  font-size: 2.5em;
-  line-height: 100%;
-  font-family: var(--header-font);
-  margin: 4rem 0px 1rem 0px;
+    font-size: 2.5em;
+    line-height: 100%;
+    font-family: var(--header-font);
+    margin: 4rem 0px 1rem 0px;
 }
 
 .centered-header {
-  font-family: var(--header-font);
-
-  position: absolute;
-  top: 40%;
-  left: 50%;
-  transform: translate(-50%, -50%);
-  text-align: center;
-  font-size: 4em;
+    font-family: var(--header-font);
+    position: absolute;
+    top: 40%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    text-align: center;
+    font-size: 4em;
 }
 
 header {
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
-  justify-content: space-between;
-  padding: 1em 0;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    padding: 1em 0;
 }
 
 header .main {
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
-  justify-content: space-between;
-  align-items: flex-start;
-  gap: 12px;
-  font-size: 1.5rem;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    align-items: flex-start;
+    gap: 12px;
+    font-size: 1.5rem;
+
+    /* Otherwise header and menu is too close on small screens*/
+    margin-bottom: 5px;
+}
+
+header .social img,
+header #dark-mode-toggle img {
+    width: 16px;
+    height: 16px;
+}
+
+header .socials {
+    margin-bottom: 10px;
+    /* Space between social icons and menu items */
+}
 
-  /* Otherwise header and menu is too close on small screens*/
-  margin-bottom: 10px;
+#dark-mode-toggle {
+    justify-content: center;
 }
 
 .socials {
-  /* flex-child */
-  flex-grow: 0;
-  /* flex-container */
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
-  justify-content: flex-start;
-  align-items: flex-end;
-  gap: 6px;
+    /* flex-child */
+    flex-grow: 0;
+    /* flex-container */
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+    align-items: flex-end;
+    gap: 6px;
 }
 
 .social {
-  border-bottom: unset;
-  background-image: unset;
-  padding: 2px;
+    border-bottom: unset;
+    background-image: unset;
+    padding: 2px;
 }
 
 .social>img {
-  border: unset;
-  width: 24px;
-  height: 24px;
+    border: unset;
+    width: 24px;
+    height: 24px;
+}
 
+/* Mobile-specific adjustments */
+@media (max-width: 600px) {
+    header {
+        flex-direction: column;
+        align-items: center;
+        padding: 1em 0;
+    }
+
+    header .main a {
+        font-size: 20px;
+    }
 }
 
 .meta {
-  color: #999;
-  letter-spacing: -0.5px;
+    color: #999;
+    display: flexbox;
+    /* This changes the meta class to use flexbox, which ensures inline display */
+    align-items: center;
+    /* Aligns items vertically in the middle */
+    flex-wrap: wrap;
+    /* Allows items to wrap as needed */
+}
+
+.meta>time {
+    color: var(--primary-color);
 }
```

Comment:
I'm not sure if I like this: 
![image](https://github.com/not-matthias/apollo/assets/26800596/2a93361d-f921-47fd-b4a5-5a426da53f27)

Would you mind changing/removing this?